sigma-impl/src/test/scala/special/sigma/BasicOpsTests.scala

```diff
@@ -18,6 +18,11 @@ class BasicOpsTests extends FunSuite with ContractsTestkit with Matchers {
     SigmaDsl.byteArrayToBigInt(collection[Byte](-1)).signum shouldBe 1
   }
 
+  test("ByteArrayToBigInt should always produce big int less than dlog group order") {
+    an [RuntimeException] should be thrownBy
+      SigmaDsl.byteArrayToBigInt(Cols.fromArray(Array.fill[Byte](500)(1)))
```

Comment:
@greenhat I suggest to add tests which also check behavior on boundary cases 
dlogGroupOrder-1, dlogGroupOrder, dlogGroupOrder+1